### DISCUSSION TITLE
Postpone to future from JobRequestHandler

### DISCUSSION THREAD

**Discussion by itizarsa** [jobrunr/jobrunr#1002] [General]
Hey folks,

I have a background job, which polls the upstream system for progress. If the operation isn't done yet, I'm creating a new job in future to check the progress. 

I'm thinking is there a way to schedule the current running job to run in future without actually creating a new job. Is it possible with Jobrunr?

**Comment by rdehuyss** [comment]
This is possible in JobRunr Pro - I have kids to feed 😉.

**Reply by itizarsa** [reply]
Still lobbying my org to buy the pro version. Hope we do that soon 🙂

**Reply by rdehuyss** [reply]
💯🙏
